doc/LIMITS.md

```diff
@@ -35,32 +35,36 @@ const node = await createLibp2pNode({
     /**
      * The total number of connections allowed to be open at one time
      */
-    maxConnections: number
+    maxConnections: number,
```

Comment:
We will have to find a way to either silence the compiler in regards to this  since `number` only refers to a type, but is being used as a value here.

I know that https://github.com/libp2p/js-libp2p/pull/1456 sort to avoid giving these values but I wonder if we should revert to stating the defaults

Reply:
It is being used as a value so the compiler is correct in failing.  To use it as a type the example needs to use an actual number (like `50`) or show the interface definition instead.

I think docs are better if the examples can be copy/pasted